### PR TITLE
DOC-213: Correct CENM log display command

### PR DESCRIPTION
**Pull request by AaronForinton**
The log display command is incorrect and requires the ```<logs-container>``` to also be included.

### CODE REVIEW

File: content/en/platform/corda/1.3/cenm/deployment-kubernetes.md [hunk 1]
```diff
@@ -293,21 +293,32 @@ You will find the truststore password in the `signer/files/pki.conf`, where the
 
 ### Display logs
 
-Each CENM service has a dedicated sidecar to display live logs from the `log/` directory.
+Each CENM service has dedicated sidecar to displays live logs from `log/` folder.
```
Comment (benpesterr3) [inline content/en/platform/corda/1.3/cenm/deployment-kubernetes.md:296]: "...to displays live logs from `log/` folder."

Is there a typo here? Should be 'to display` no?

also why folder not directory?

File: content/en/platform/corda/1.4/cenm/deployment-kubernetes.md [hunk 2]
```diff
@@ -294,21 +294,31 @@ You will find the truststore password in the `signer/files/pki.conf`, where the
 
 ### Display logs
 
-Each CENM service has a dedicated sidecar to display live logs from the `log/` directory.
+Each CENM service has dedicated sidecar to displays live logs from `log/` folder.
```
Comment (benpesterr3) [inline content/en/platform/corda/1.4/cenm/deployment-kubernetes.md:297]: displays = display
folder or directory?

File: content/en/platform/corda/1.5/cenm/deployment-kubernetes.md [hunk 3]
```diff
@@ -299,21 +299,31 @@ You will find the truststore password in the `signer/files/pki.conf`, where the
 
 ### Display logs
 
-Each CENM service has a dedicated sidecar to display live logs from the `log/` directory.
+Each CENM service has dedicated sidecar to displays live logs from `log/` folder.
```
Comment (benpesterr3) [inline content/en/platform/corda/1.5/cenm/deployment-kubernetes.md:302]: Displays = display
folder/directory?

**Review by benpesterr3 — COMMENTED** [review]
some comments and I think a typo

File: content/en/platform/corda/1.2/cenm/deployment-kubernetes.md [hunk 4]
```diff
@@ -275,18 +275,29 @@ For more details about joining CENM network see: [Joining an existing compatibil
...
 ### Display logs
 
-Each CENM service has dedicated sidecar to displays live logs from `log/` folder.
+Each CENM service has a dedicated sidecar to displays live logs from the `log/` directory.
 
 To display logs use the following command:
 
 ```bash
-kubectl logs -c logs <pod-name>
+kubectl logs -c logs <logs-container> <pod-name>
```
Comment (michelemarsden) [inline content/en/platform/corda/1.2/cenm/deployment-kubernetes.md:283]:
```suggestion
kubectl logs -c <logs-container> <pod-name>
```

File: content/en/platform/corda/1.2/cenm/deployment-kubernetes.md [hunk 5]
```diff
@@ -275,18 +275,29 @@ For more details about joining CENM network see: [Joining an existing compatibil
...
+kubectl logs -c logs-nmap <pod-name>   //for nmap
+kubectl logs -c logs-signer <pod-name>   //for signer
+kubectl logs -c logs-zone <pod-name>   //for zone
 ```
 
 To display live logs use the following command:
 
 ```bash
-kubectl logs -c logs -f <pod-name>
+kubectl logs -c logs -f <logs-container> <pod-name>
```
Comment (michelemarsden) [inline content/en/platform/corda/1.2/cenm/deployment-kubernetes.md:300]: does it need the 2nd logs?

File: content/en/platform/corda/1.2/cenm/deployment-kubernetes.md [hunk 6]
```diff
@@ -275,18 +275,29 @@ For more details about joining CENM network see: [Joining an existing compatibil
...
+
+The ```<logs-container>``` object container determines where the logs will be drawn from. The services write their logs to dedicated display containers
+where you can get a live view of the logs:
+```bash
+kubectl logs -c logs-auth <pod-name>   //for auth
+kubectl logs -c logs-gateway <pod-name>   //for gateway
+kubectl logs -c logs-idman <pod-name>   //for idman
+kubectl logs -c logs-nmap <pod-name>   //for nmap
+kubectl logs -c logs-signer <pod-name>   //for signer
+kubectl logs -c logs-zone <pod-name>   //for zone
```
Comment (michelemarsden) [inline content/en/platform/corda/1.2/cenm/deployment-kubernetes.md:294]: Are you adding in: 
//For notary/pki/hsm is slightly different as it doesn't have a logs-notary container
kubectl logs <pod-name>   //for notary, pki, hsm

**Review by michelemarsden — COMMENTED** [review]
I've only made comments on one file, but those would apply to the others. Can you double-check with an SME please?

File: content/en/platform/corda/1.2/cenm/deployment-kubernetes.md [hunk 7]
```diff
@@ -275,18 +275,32 @@ For more details about joining CENM network see: [Joining an existing compatibil
...
+```
+For notary, PKI and HSM the command is slightly different as logs containers do not exist for these services:
```
Comment (michelemarsden) [inline content/en/platform/corda/1.2/cenm/deployment-kubernetes.md:296]:
```suggestion
For notary, PKI, and HSM, the command is slightly different as logs containers do not exist for these services:
```

File: content/en/platform/corda/1.2/cenm/deployment-kubernetes.md [hunk 8]
```diff
@@ -275,18 +275,32 @@ For more details about joining CENM network see: [Joining an existing compatibil
...
-kubectl logs -c logs <pod-name>
+kubectl logs -c <logs-container> <pod-name>
 ```
 
+The ```<logs-container>``` object container determines where the logs will be drawn from. The services write their logs to dedicated display containers
+where you can get a live view of the logs:
+```bash
+kubectl logs -c logs-auth <pod-name>   //for auth
+kubectl logs -c logs-gateway <pod-name>   //for gateway
+kubectl logs -c logs-idman <pod-name>   //for idman
```
Comment (michelemarsden) [inline content/en/platform/corda/1.2/cenm/deployment-kubernetes.md:291]:
```suggestion
kubectl logs -c logs-idman <pod-name>   //for identity manager
```
did we want to say identity manager?

File: content/en/platform/corda/1.2/cenm/deployment-kubernetes.md [hunk 9]
```diff
@@ -275,18 +275,32 @@ For more details about joining CENM network see: [Joining an existing compatibil
...
+kubectl logs -c logs-nmap <pod-name>   //for nmap
```
Comment (michelemarsden) [inline content/en/platform/corda/1.2/cenm/deployment-kubernetes.md:292]:
```suggestion
kubectl logs -c logs-nmap <pod-name>   //for network map
```
Did we want to say network map?

File: content/en/platform/corda/1.3/cenm/deployment-kubernetes.md [hunk 10]
```diff
@@ -293,21 +293,36 @@ You will find the truststore password in the `signer/files/pki.conf`, where the
...
+where you can get a live view of the logs:
+```bash
+kubectl logs -c logs-auth <pod-name>   //for auth
+kubectl logs -c logs-gateway <pod-name>   //for gateway
+kubectl logs -c logs-idman <pod-name>   //for idman
+kubectl logs -c logs-nmap <pod-name>   //for nmap
+kubectl logs -c logs-signer <pod-name>   //for signer
+kubectl logs -c logs-zone <pod-name>   //for zone
+```
+For notary, PKI and HSM the command is slightly different as logs containers do not exist for these services:
```
Comment (michelemarsden) [inline content/en/platform/corda/1.3/cenm/deployment-kubernetes.md:314]:
```suggestion
For notary, PKI, and HSM, the command is slightly different as logs containers do not exist for these services:
```

File: content/en/platform/corda/1.3/cenm/deployment-kubernetes.md [hunk 11]
```diff
@@ -293,21 +293,36 @@ You will find the truststore password in the `signer/files/pki.conf`, where the
...
+kubectl logs -c <logs-container> <pod-name>
+```
+
+The ```<logs-container>``` object container determines where the logs will be drawn from. The services write their logs to dedicated display containers
+where you can get a live view of the logs:
+```bash
+kubectl logs -c logs-auth <pod-name>   //for auth
+kubectl logs -c logs-gateway <pod-name>   //for gateway
+kubectl logs -c logs-idman <pod-name>   //for idman
+kubectl logs -c logs-nmap <pod-name>   //for nmap
```
Comment (michelemarsden) [inline content/en/platform/corda/1.3/cenm/deployment-kubernetes.md:310]:
```suggestion
kubectl logs -c logs-idman <pod-name>   //for identity manager
kubectl logs -c logs-nmap <pod-name>   //for network map
```

File: content/en/platform/corda/1.4/cenm/deployment-kubernetes.md [hunk 12]
```diff
@@ -294,21 +294,35 @@ You will find the truststore password in the `signer/files/pki.conf`, where the
...
+```
 
-Use the following command to display live logs:
+The ```<logs-container>``` object container determines where the logs will be drawn from. The services write their logs to dedicated display containers
+where you can get a live view of the logs:
+```bash
+kubectl logs -c logs-auth <pod-name>   //for auth
+kubectl logs -c logs-gateway <pod-name>   //for gateway
+kubectl logs -c logs-idman <pod-name>   //for idman
+kubectl logs -c logs-nmap <pod-name>   //for nmap
```
Comment (michelemarsden) [inline content/en/platform/corda/1.4/cenm/deployment-kubernetes.md:311]:
```suggestion
kubectl logs -c logs-idman <pod-name>   //for identity manager
kubectl logs -c logs-nmap <pod-name>   //for network map
```

File: content/en/platform/corda/1.4/cenm/deployment-kubernetes.md [hunk 13]
```diff
@@ -294,21 +294,35 @@ You will find the truststore password in the `signer/files/pki.conf`, where the
...
+kubectl logs -c logs-signer <pod-name>   //for signer
+kubectl logs -c logs-zone <pod-name>   //for zone
+```
+For notary, PKI and HSM the command is slightly different as logs containers do not exist for these services:
```
Comment (michelemarsden) [inline content/en/platform/corda/1.4/cenm/deployment-kubernetes.md:315]:
```suggestion
For notary, PKI, and HSM, the command is slightly different as logs containers do not exist for these services:
```

File: content/en/platform/corda/1.5/cenm/deployment-kubernetes.md [hunk 14]
```diff
@@ -299,21 +299,36 @@ You will find the truststore password in the `signer/files/pki.conf`, where the
...
+kubectl logs -c <logs-container> <pod-name>
+```
+
+The ```<logs-container>``` object container determines where the logs will be drawn from. The services write their logs to dedicated display containers
+where you can get a live view of the logs:
+```bash
+kubectl logs -c logs-auth <pod-name>   //for auth
+kubectl logs -c logs-gateway <pod-name>   //for gateway
+kubectl logs -c logs-idman <pod-name>   //for idman
+kubectl logs -c logs-nmap <pod-name>   //for nmap
```
Comment (michelemarsden) [inline content/en/platform/corda/1.5/cenm/deployment-kubernetes.md:316]:
```suggestion
kubectl logs -c logs-idman <pod-name>   //for identity manager
kubectl logs -c logs-nmap <pod-name>   //for network map
```

File: content/en/platform/corda/1.5/cenm/deployment-kubernetes.md [hunk 15]
```diff
@@ -299,21 +299,36 @@ You will find the truststore password in the `signer/files/pki.conf`, where the
...
+kubectl logs -c logs-signer <pod-name>   //for signer
+kubectl logs -c logs-zone <pod-name>   //for zone
+```
+For notary, PKI and HSM the command is slightly different as logs containers do not exist for these services:
```
Comment (michelemarsden) [inline content/en/platform/corda/1.5/cenm/deployment-kubernetes.md:320]:
```suggestion
For notary, PKI, and HSM, the command is slightly different as logs containers do not exist for these services:
```

**Review by michelemarsden — COMMENTED** [review]
Minor edits.

**Review by michelemarsden — APPROVED** [review]
LGTM

**Pull request merged**